internal/controller/database_controller.go

```diff
@@ -176,70 +168,89 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	// database status not true, process phase
-	if !dbcr.Status.Status {
-		ownership := []metav1.OwnerReference{}
-		if dbcr.Spec.Cleanup {
-			ownership = append(ownership, metav1.OwnerReference{
-				APIVersion: dbcr.APIVersion,
-				Kind:       dbcr.Kind,
-				Name:       dbcr.Name,
-				UID:        dbcr.GetUID(),
-			},
-			)
-		}
-
-		phase := dbcr.Status.Phase
-		logrus.Infof("DB: namespace=%s, name=%s start %s", dbcr.Namespace, dbcr.Name, phase)
-
-		defer promDBsPhaseTime.WithLabelValues(phase).Observe(kci.TimeTrack(time.Now()))
-		err := r.createDatabase(ctx, dbcr, ownership)
-		if err != nil {
-			// when database creation failed, don't requeue request. to prevent exceeding api limit (ex: against google api)
-			return r.manageError(ctx, dbcr, err, false)
-		}
-		dbcr.Status.Phase = dbPhaseInstanceAccessSecret
 
-		if err = r.createInstanceAccessSecret(ctx, dbcr, ownership); err != nil {
-			return r.manageError(ctx, dbcr, err, true)
-		}
-		dbcr.Status.Phase = dbPhaseProxy
-		err = r.createProxy(ctx, dbcr, ownership)
-		if err != nil {
-			return r.manageError(ctx, dbcr, err, true)
-		}
-		dbcr.Status.Phase = dbPhaseSecretsTemplating
-		if err = r.createTemplatedSecrets(ctx, dbcr, ownership); err != nil {
-			return r.manageError(ctx, dbcr, err, true)
-		}
-		dbcr.Status.Phase = dbPhaseConfigMap
-		if err = r.createInfoConfigMap(ctx, dbcr, ownership); err != nil {
-			return r.manageError(ctx, dbcr, err, true)
-		}
-		dbcr.Status.Phase = dbPhaseTemplating
-		// A temporary check that exists to avoid creating templates if secretsTemplates are used.
-		// todo: It should be removed when secretsTemlates are gone
-		if len(dbcr.Spec.SecretsTemplates) == 0 {
-			if err := r.renderTemplates(ctx, dbcr); err != nil {
+	phase := dbcr.Status.Phase
```

Comment:
would be `nil` at the beginning, or?

Reply:
I'm not sure which line you're talking about here